### PR TITLE
Removes VoGs ability to force people to cum

### PR DESCRIPTION
**Pull request by MrJWhit**
## About The Pull Request

Mirror of: https://github.com/Skyrat-SS13/Skyrat13/pull/2894

## Why It's Good For The Game

Removes the creepy ability to mass force people to cum without consent. This has been around for too long and should get cleaned up.

## Changelog
:cl:
del: Removed VoG forcing cumming
/:cl:

### CODE REVIEW

**Comment by HeroWithYay** [comment]
Yeah fair

**Comment by Ludox235** [comment]
Speedmerge now

**Comment by silicons** [comment]
@MrJWhit fix pr, you touched changelogs for some rason

**Comment by MrJWhit** [comment]
Well first I need to figure out wtf is going on with my changelogs, even if I need to update it all manually. Why is life hard.

File: code/modules/surgery/organs/vocal_cords.dm [hunk 1]
```diff
@@ -254,7 +254,7 @@
 	var/static/regex/clap_words = regex("clap|applaud")
 	var/static/regex/honk_words = regex("ho+nk") //hooooooonk
 	var/static/regex/multispin_words = regex("like a record baby|right round")
-	var/static/regex/orgasm_words = regex("cum|orgasm|climax|squirt|heyo") //CITADEL CHANGE
+	// var/static/regex/orgasm_words = regex("cum|orgasm|climax|squirt|heyo") //Removed from cit later
```
Comment (silicons) [inline code/modules/surgery/organs/vocal_cords.dm]: also dude just toss this don't leave it commented

**Comment by Tupinambis** [comment]
Yeah this doesn't really add anything besides ranging from unfunny meme to downright creepy. Good PR.

**Comment by Hatterhat** [comment]
twenty seven thousand line changes

**Comment by Ludox235** [comment]
WHY DID YOU DELETE CHANGELOG STUFF

**Comment by MrJWhit** [comment]
My changelog stuff was fucked since last time, something to do with how my basic bitch workflow goes. I'm talking to kevinz in how to fix it.

**Comment by Hatterhat** [comment]
too late i already did the same thing out of spite

**Comment by MrJWhit** [comment]
Cool, time to figure out wtf happened to my github.
Also, make sure to give credit :v)

**Pull request closed without merging**